transport/src/main/java/io/netty/channel/AbstractChannel.java

```diff
@@ -349,7 +349,7 @@ public String toString() {
                 srcAddr = remoteAddr;
                 dstAddr = localAddr;
```

Comment:
Swapping addresses in the string representation can be confusing for datagram sockets and P2P sockets, so I'd like to remove swapping completely.